.github/workflows/plugins-jdk14-test.0.yaml

```diff
@@ -62,11 +62,13 @@ jobs:
       - uses: actions/checkout@v2
         with:
           submodules: true
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-          java-version: 14
+          # temurin does not distribute non-lts version
+          distribution: 'zulu'
```

Comment:
Why do you use these distribution instead of those that we are using in our official Docker images?